docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -509,12 +510,33 @@ If you're upgrading to a new version, the upgrade process depends on the deploym
 - [Upgrading&#8212;Terraform Template](#upgradingterraform-template)
 - [Upgrading&#8212;gcloud CLI](#upgradinggcloud-cli)
 
-### Upgrading&#8212;Terraform Template
+#### Upgrading&#8212;Terraform Template
 
 If you deployed using the Terraform template, all you need to do to upgrade is update your deployment with the new `{OPERATOR_IMAGE}` that you received in the upgrade notification.
 
-### Upgrading&#8212;gcloud CLI
+#### Upgrading&#8212;gcloud CLI
 
 If you deployed using the gcloud CLI, you must manually bring up new instances that use the new `{OPERATOR_IMAGE}` and then shut down old instances.
 
 If you previously set up a load balancer manually, you'll also need to update the mapping for the load balancer.
+
+## Scraping Metrics
+This section provides information on the process of scraping metrics. 
+
+We expose 9080 port which serves Prometheus metrics (/metrics). 
```

Comment:
We provide prometheus format metrics on port 9080. You can use prometheus compatible scraper to collect and aggregate these metrics.